app/controllers/grant_submissions/submissions_controller.rb

```diff
@@ -55,6 +55,7 @@ def create
       if @submission.save
         @submission.update(user_updated_at: Time.now)
         if @submission.submitted?
+          send_grant_admin_submission_notification
```

Comment:
Should this be conditional -- i.e. if anyone is supposed to receive notifications

Reply:
Right now that is handled inside the mailer. It searches for the emails and then if the emails are empty, it doesn't create the mailer. 

It could potentially be pulled out here.

Reply:
Yeah, it might be good to avoid the method if it's not needed